cypress/e2e/permalinks/permalinks.spec.ts

```diff
@@ -131,6 +131,7 @@ describe("permalinks", () => {
         cy.get(".mx_NewRoomIntro").invoke("remove");
         cy.get(".mx_GenericEventListSummary").invoke("remove");
 
-        cy.get(".mx_RoomView_timeline").percySnapshotElement("Permalink rendering");
+        const percyCSS = ".mx_MessageTimestamp, .mx_RoomView_myReadMarker { visibility: hidden !important; }";
```

Comment:
This will have to be updated once https://github.com/matrix-org/matrix-react-sdk/pull/10745 is merged.